paket.dependencies

```diff
@@ -0,0 +1,12 @@
+framework auto-detect
+
+source https://www.nuget.org/api/v2
+
+nuget Autofac ~> 4.8

```

Comment:
Does this project need all this packets?

Reply:
ja

Reply:
I can only see following dependencies
`FileOpeningBuilder\paket.references` does not need any pakets
`FileOpeningBuilder.Test\paket.references` does only need `using Messerli.Test.Utility`

Reply:
![image](https://user-images.githubusercontent.com/32902523/56950930-d8ba3a00-6b36-11e9-8d9f-6c6be92d050f.png)

![image](https://user-images.githubusercontent.com/32902523/56950965-e7a0ec80-6b36-11e9-8398-50b290ebe74b.png)

Reply:
True but those files souldn't contain all of the pakets i.e autofac or JetBrains.Annotations

Reply:
ac89b63d2e16f9159c606ccbb9896015b6bccc8b